.github/workflows/codeql-workflow.yml

```diff
@@ -39,8 +39,7 @@ jobs:
       # CodeQL will create the database during the compilation
       - name: Build
         run: |
-          git config --global --add safe.directory "$GITHUB_WORKSPACE"
```

Comment:
This command is now directly performed by the Docker image build, right ?

Reply:
Yes IF the github action is `checkout@v3`